jerry-core/ecma/base/ecma-lcache.c

```diff
@@ -72,7 +65,7 @@ ecma_lcache_row_index (jmem_cpointer_t object_cp, /**< compressed pointer to obj
 {
   /* Randomize the hash of the property name with the object pointer using a xor operation,
    * so properties of different objects with the same name can be cached effectively. */
-  return (size_t) ((name_hash ^ object_cp) & ECMA_LCACHE_HASH_MASK);
+  return (size_t) ((name_hash ^ object_cp) & (ECMA_LCACHE_HASH_ROWS_COUNT - 1));
```

Comment:
Is there an assert somewhere which checks that ECMA_LCACHE_HASH_ROWS_COUNT is a power of 2?